ProductPOC/appsettings.json

```diff
@@ -5,5 +5,12 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "MongoDb": "mongodb+srv://sauravbalasahebchoudhari:37JoCLVY7OSMX3dc@cluster0.lyiyfwj.mongodb.net/"
```

Comment:
You can use user-secrets, avoid committing it to git

Reply:
updated code with user-secrets.

Reply:
I can still see the connection string , are these changes in another PR?

Reply:
yes check latest PR 3 in which i updated.

Reply:
Ideally you are supposed to resolve in this PR so that we can merge this.

Reply:
But fine, you can merge rest PRs into this branch, so we would have one PR with all fixes

Reply:
yes,  PR3 is having code of PR1 and PR2 also.